spec/models/job_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Job, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "#instantiate" do
```

Comment:
This context should be called 'New', only use double quotes if you are interpolating, and should include the other example as well. You would then only need a single let declaration for job.